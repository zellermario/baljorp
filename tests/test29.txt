create SJ kezdes 0
create ISJ vegzes 0 2
addNeighbor kezdes 1 vegzes
addNeighbor vegzes 2 kezdes

create Eszkimo e vegzes
create jegesmedve m kezdes

move m 1
pass m