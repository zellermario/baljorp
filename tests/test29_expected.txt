Fields:
SJ kezdes (0) - object: x
	neighbours: [(vegzes, 1)]
ISJ vegzes (0,2) - object: x
	neighbours: [(kezdes, 2)]

Buildings:

Actors:
> Eszkimo eszkimo (heat: 0) on vegzes
  Inventory: []
> Medve m on vegzes
